backend/scripts/curation/scv2/curate.py

```diff
@@ -0,0 +1,151 @@
+    del adata.obs["scv2_10+"]
+    adata.obs["Infected"][adata.obs["Condition"] == "Mock"] = "Mock"
+
+    adata.obs["Cell type"] = adata.obs["ctype"]
+    del adata.obs["ctype"]
+
+    adata.uns["contributors"] = [
+        {"name": "Neal G. Ravindra", "institution": "Yale University", "email": "neal.ravindra@yale.edu"}
+    ]
+    adata.uns["preprint_doi"] = "https://doi.org/10.1101/2020.05.06.081695"
+
+
+def remix(adata):
+    """Create the full Corpora remix"""
+
+    # First fill in missing metadata fields
+    adata.obs["assay_ontology"] = "EFO:0009922"
+    adata.obs["assay"] = utils.ontology.get_ontology_label("EFO:0009922")
+
+    disease_ontology_map = {
+        "1dpi": "MONDO:0100096",
+        "2dpi": "MONDO:0100096",
+        "3dpi": "MONDO:0100096",
+        "Mock": "PATO:0000461",
+    }
+    disease_map = {k: utils.ontology.get_ontology_label(v) for k, v in disease_ontology_map.items()}
+
+    adata.obs["disease_ontology"] = adata.obs["Condition"].replace(disease_ontology_map, inplace=False)
+    adata.obs["disease"] = adata.obs["Condition"].replace(disease_map, inplace=False)
+
+    adata.obs["tissue_ontology"] = "UBERON:0002031 (cell culture)"
+    adata.obs["tissue"] = f"{utils.ontology.get_ontology_label('UBERON:0002031')} (cell culture)"
+
+    adata.uns["organism_ontology"] = "NCBITaxon:9606"
+    adata.uns["organism"] = utils.ontology.get_ontology_label("NCBITaxon:9606")
+    adata.uns["title"] = "Single-cell longitudinal analysis of SARS-CoV-2 infection in human bronchial epithelial cells"
+
+    adata.uns[
+        "project_name"
+    ] = "Single-cell longitudinal analysis of SARS-CoV-2 infection in human bronchial epithelial cells"
+    adata.uns["project_description"] = (
+        "Single-cell RNA sequencing of experimentally infected human bronchial epithelial "
+        "cells (HBECs) in air-liquid interface cultures over a time-course"
+    )
+    adata.uns["project_other_links"] = ["https://github.com/vandijklab/HBEC_SARS-CoV-2_scRNA-seq"]
+    # Set the cell ontology values
+    # BC/Club is "a cell population intermediate between basal cells and club cells". There is no
+    # CL term for them.
+
+    cell_type_ontology_map = {
+        "Basal cells": "CL:1000349",
+        "Ciliated cells": "CL:0002332",
+        "Club cells": "CL:0000158",
+        "BC/Club": "",
+        "Ionocytes": "CL:0017000",
+        "Neuroendocrine cells": "CL:1000223",
+        "Tuft cells": "CL:0002208",
+        "Goblet cells": "CL:1000312",
+    }
+    cell_type_map = {k: utils.ontology.get_ontology_label(v) for k, v in cell_type_ontology_map.items()}
+    cell_type_map["BC/Club"] = "BC/Club"
+
+    adata.obs["cell_type_ontology"] = adata.obs["Cell type"].replace(cell_type_ontology_map)
+    adata.obs["cell_type"] = adata.obs["Cell type"].replace(cell_type_map)
+    del adata.obs["Cell type"]
+
+    # Now translate the gene symbols and sum new duplicates
```

Comment:
Given that `upgrade_symbol()` produces duplicate symbols, summing raw counts and normalizing will be a requirement for datasets that require gene symbol conversion. Is this something we're prepared to do?

Reply:
Yeah, I think we're prepared to do that unenthusiastically. There's an example of that in the scv2 script. A pretty common update to HGNC is saying gene A is really just a part of gene B, so A gets merged into B. So I think we need to handle those cases or we'll be missing a lot.